wrappers/python/simtk/openmm/__init__.py

```diff
@@ -20,6 +20,8 @@
 
 from simtk.openmm.openmm import *
 from simtk.openmm.vec3 import Vec3
+from mtsintegrator import MTSIntegrator
+from amd import AMDIntegrator, AMDForceGroupIntegrator, DualAMDIntegrator
```

Comment:
*implicit* relative imports that is.  `from .mtsintegrator import MTSIntegrator` I think is valid.  Maybe I misunderstood PEP 328.